x/oracle/keeper/msg_server.go

```diff
@@ -125,19 +106,6 @@ func (ms msgServer) AggregateExchangeRateVote(
 	ms.SetAggregateExchangeRateVote(ctx, valAddr, types.NewAggregateExchangeRateVote(filteredTuples, valAddr))
 	ms.DeleteAggregateExchangeRatePrevote(ctx, valAddr)
 
-	ctx.EventManager().EmitEvents(sdk.Events{
```

Comment:
-

Reply:
same, this event doesn't bring anything. We are emitting events when we set the vote (during the event blocker), not during the voting.